contracts/ICS26Router.sol

```diff
@@ -126,6 +129,10 @@ contract ICS26Router is
         require(
             msg_.timeoutTimestamp > block.timestamp, IBCInvalidTimeoutTimestamp(msg_.timeoutTimestamp, block.timestamp)
         );
+        require(
+            msg_.timeoutTimestamp - block.timestamp <= MAX_TIMEOUT_DURATION,
+            IBCInvalidTimeoutDuration(msg_.timeoutTimestamp - block.timestamp, MAX_TIMEOUT_DURATION)
```

Comment:
arg order is switched here

Reply:
This order feels more natural, so perhaps switch the order in the error itself?

Reply:
I will fix the require statement now. The order of the error should remain the same because in other errors we always put the expected value first and then the actual value.